src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Negotiate/NegotiateStream_Http_Tests.cs

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Infrastructure.Common;
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Security;
+using Xunit;
+
+public static class NegotiateStream_Http_Tests
+{
+    // The tests are as follows:
+    //
+    // NegotiateStream_*_AmbientCredentials
+    //     Windows: This should pass by default without any code changes
+    //       Linux: This should not pass by default 
+    //              Run 'kinit user@DC.DOMAIN.COM' before running this test to use ambient credentials
+    //              ('DC.DOMAIN.COM' must be in capital letters) 
+    //              If previous tests were run, it may be necessary to run 'kdestroy -A' to remove all
+    //              prior Kerberos tickets
+    // 
+    // NegotiateStream_*_With_ExplicitUserNameAndPassword
+    //     Windows: Edit the s_UserName and s_Password variables to a user valid on your Kerberos realm
+    //       Linux: Edit the s_UserName and s_Password variables to a user valid on your Kerberos realm
```

Comment:
I like your proposal to use TestProperties, and think all the instructions that say "edit xyz..." should just call out the TestProperty name.  We could also use [ConditionalFact] to skip these tests automatically if the TestProperties required are not specified.